flink-contrib/flink-storm-examples/src/main/java/org/apache/flink/storm/util/FileSpout.java

```diff
@@ -38,6 +38,8 @@
 	protected String path = null;
 	protected BufferedReader reader;
 
+	protected boolean finished;
+
 	public FileSpout() {}
```

Comment:
The point it the following. `FileSpout` is a Flink agnostic implementation that is improved in a Flink aware way by `FiniteFileSpout`. Thus, `FileSpout` should be implemented the Storm way without any knowledge of Flink. And `FiniteFileSpout` should use Flink stuff, but not change the code of `FileSpout`.